predictions.py

```diff
@@ -4,6 +4,12 @@
 based prediction routines.
 """
 
+import storage
+import os
+
+# e.g. MODEL_STORE_URI=mongodb://localhost:27017
+MODEL_STORE_URI = os.environ['MODEL_STORE_URI']
```

Comment:
i'm ok with leaving static variables at the top level, but something evaluated like this should be done where it is used or wrapped with a function. just to avoid any possible side effects from importing the file.